### DISCUSSION TITLE
Can change HOME button to specify folder?

### DISCUSSION THREAD

**Discussion by SimonChiu930** [rejetto/hfs#752] [Q&A] [answered]
I want open the index.html first and redirect to specify home folder, Can i do that?
(because re-open the index page when click the HOME button)

![WeChat Screenshot_20240926164803](https://github.com/user-attachments/assets/16d879c6-d56d-4d2d-a9c7-270e4cd3b749)

**Comment by rejetto** [comment]
the best way to do this is to go Admin-panel > internet > address
and add a row with domain `*` and home `/Folders/`
<img width="672" alt="image" src="https://github.com/user-attachments/assets/182f2b49-85f9-4a93-9046-9b3d182e1b06">

**Reply by SimonChiu930** [reply]
this action default will go to www.abc.com\Folders

**Comment by rejetto** [comment]
i missed the part where you asked for "index.html"
let me think

**Reply by SimonChiu930** [reply]
example:
Shared Files > VFS:
Home, Disk Source: C:\Web (index.html), enable "Serve as web-page if index.html is found"
under Home, Folders, Disk Source: C:\Folders (have a sub-folder)

when I open www.abc.com, will show the index page first, and than redirect to www.abc.com/folders after 5sec.
(but this action, the Home button will loop open the index page)

**Comment by rejetto** [comment]
so, your problem is that you dont' want the user to be able to click home to go back to the index.html ?
then just hide the home button using css

**Reply by SimonChiu930** [reply]
Yes, my limited English skills make it difficult for you to understand, thanks for the reply.

**Comment by rejetto** [comment, marked as answer]
admin-panel > custom html > section style

`.breadcrumb:nth-of-type(2) { display: none }`

**Reply by SimonChiu930** [reply]
I'm off work now, waiting to test tomorrow.

**Reply by SimonChiu930** [reply]
work now, thanks!
